integration-tests/src/test/kotlin/org/stellar/anchor/platform/AnchorPlatformIntegrationTest.kt

```diff
@@ -94,10 +93,10 @@ class AnchorPlatformIntegrationTest {
     val response = SepClient.client.newCall(request).execute()
     val responseBody = response.body?.string()
     assertNotNull(responseBody)
-
-    val gotHealthCheck = SepClient.gson.fromJson(responseBody, HealthCheck::class.java)
-    val wantHealthCheck = HealthCheck(true)
-    assertEquals(wantHealthCheck, gotHealthCheck)
+    assertEquals(200, response.code)
+//    val gotHealthCheck = SepClient.gson.fromJson(responseBody, HashMap::class.java)
+//    assertNotNull(gotHealthCheck["number_of_checks"])
+//    assertNotNull(gotHealthCheck["checks"])
```

Comment:
❗  Could you add some test here?

We're deploying `/health` as a way for other services to check if Platform is up, so it makes sense to me that we have at least smoke tests to make sure it is working and exposing the info as expected. This way we can guarantee some reliability to this endpoint others will be relying on.

Reply:
The tests are actually quite tricky because `Gson` throws the following exception while we call 
```java
gson.fromJson(responseBody, HealthCheckResponse::class.java)
```

Exception:
```
Caused by: java.lang.UnsupportedOperationException: Interface can't be instantiated! Interface name: org.stellar.anchor.api.platform.HealthCheckResult
	at com.google.gson.internal.UnsafeAllocator.assertInstantiable(UnsafeAllocator.java:117)
	at com.google.gson.internal.UnsafeAllocator$1.newInstance(UnsafeAllocator.java:49)
	at com.google.gson.internal.ConstructorConstructor$14.construct(ConstructorConstructor.java:225)
	... 80 more
```
That was the reason they were commented out. 

Nevertheless, I had added some tedious, verbose but necessary tests.